### PR TITLE
Fix/UI update checkout

### PR DESCRIPTION
**Pull request by rhimmelbauer**
Notes:
- Checkbox of same as shipping address is pre set to true
- Alignment fix for forms.
- Leading zero for single digit months

### CODE REVIEW

**Review by mgcamisa — APPROVED** [review]

File: vendor/templates/vendor/checkout.html [hunk 1]
```diff
@@ -202,10 +202,13 @@ <h6 class="my-0">{{ item.name }}</h6>
 
 {% block extra_js %}
 <script>
-  $('#id_same_as_shipping').prop("checked", false);
+  $('#id_same_as_shipping').prop("checked", true);
```
Comment (devo-wm) [inline vendor/templates/vendor/checkout.html:205]: if you have this set as the initial value I wonder if you even need this line of js

Comment (devo-wm) [reply vendor/templates/vendor/checkout.html:205]: snap I wonder if just none of these should be required on load; and then you just validate on the server side in the form clean method: https://docs.djangoproject.com/en/3.1/ref/forms/validation/#validating-fields-with-clean

Comment (rhimmelbauer) [reply vendor/templates/vendor/checkout.html:205]: the thing is that if you do not set it always on refresh it saves that last state so if you uncheck and don't force the checked value the switch between the fields and the shipping address gets messed up

Comment (rhimmelbauer) [reply vendor/templates/vendor/checkout.html:205]: I would leave them required in the form, less work to do, work that Django already those for us. Removing the required in the html allows for the submit to get posted.

Comment (devo-wm) [reply vendor/templates/vendor/checkout.html:205]: 👍

**Review by devo-wm — APPROVED** [review]

**Pull request merged**